lib/opal/app_creator.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+require 'tmpdir'
+require 'opal/os'
+
+module Opal
+  class AppCreator
```

Comment:
Here it's still called AppCreator and not something like ExeBuilder

Reply:
Oh, sure, i forgot to change the class name. Next time then.